### ISSUE TITLE
Build Failures since GMEXT-AdMob 1.4.0 with Android

### ISSUE THREAD

**Issue by KBeeeR** [YoYoGames/GMEXT-AdMob#72]
Hello,

I’m encountering build failures when targeting Android since GMEXT-AdMob version 1.4.0. The same issue persists in 1.4.1 and 1.4.2, but the project builds successfully with 1.3.11. Below are the details and error log:

Environment:

• GameMaker: v2024.8.1.218
• Android API Level: 35 & 34
• AdMob SDK: 1.4.0, 1.4.1, 1.4.2 (issue), 1.3.11, 1.3.10 (works)

Error log:
```
> Task :com.company.game:compileDebugJavaWithJavac FAILED
C:\Users\user\AppData\Roaming\GameMakerStudio2\Cache\GMS2CACHE\AdMob_427A8D22\android\Default\com.company.game\src\main\java\com\company\game\GoogleMobileAdsGM.java:1785: error: cannot find symbol
                        if (Math.abs(longValue) <= MAX_DOUBLE_SAFE) {
                                                   ^
  symbol:   variable MAX_DOUBLE_SAFE
  location: class GoogleMobileAdsGM
C:\Users\user\AppData\Roaming\GameMakerStudio2\Cache\GMS2CACHE\AdMob_427A8D22\android\Default\com\company\game\src\main\java\com\company\game\GoogleMobileAdsGM.java:1792: error: cannot find symbol
                        String jsonString = new JSONObject((Map) value).toString();
                                                ^
  symbol:   class JSONObject
  location: class GoogleMobileAdsGM
C:\Users\user\AppData\Roaming\GameMakerStudio2\Cache\GMS2CACHE\AdMob_427A8D22\android\Default\com\company\game\src\main\java\com\company\game\GoogleMobileAdsGM.java:1795: error: cannot find symbol
                        String jsonString = new JSONArray((List) value).toString();
                                                ^
  symbol:   class JSONArray
  location: class GoogleMobileAdsGM
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
3 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':com.company.game:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
``` 

Please advise on a solution or provide guidance on how to resolve this issue.

Thank you!

**Comment by DiasFranciscoA** [comment]
Fixed in 1.4.2 (Hotfix) will be 1.4.2.1 on Marketplace

**Comment by KBeeeR** [comment]
Tested and it’s working. Thank you!
